shell/platform/windows/flutter_windows_view.cc

```diff
@@ -10,6 +10,10 @@ namespace flutter {
 
 FlutterWindowsView::FlutterWindowsView(
     std::unique_ptr<WindowBindingHandler> window_binding) {
+  resize_status_ = ResizeState::kDone;
+  resize_target_width_ = 0;
+  resize_target_height_ = 0;
```

Comment:
Since we're using modern C++, prefer initializing ivars at the declaration point whenever possible (which is the case for all three of these).